### PR TITLE
Translator function 0.0.1

### LINKED ISSUE #1: Implement Translator Function
We need function to translate forward and backward between *text* and *morse code*

I have this object to help you implement this function easier!
```js
const morse = {
  A: '.-', B: '-...', C: '-.-.', D: '-..', E: '.', F: '..-.', G: '--.', H: '....',
  I: '..', J: '.---', K: '-.-', L: '.-..', M: '--', N: '-.', O: '---', P: '.--.',
  Q: '--.-', R: '.-.', S: '...', T: '-', U: '..-', V: '...-', W: '.--', X: '-..-',
  Y: '-.--', Z: '--..', 0: '-----', 1: '.----', 2: '..---', 3: '...--', 4: '....-',
  5: '.....', 6: '-....', 7: '--...', 8: '---..', 9: '----.', '.': '.-.-.-', ',': '--..--',
  '?': '..--..', "'": '.----.', '!': '-.-.--', '/': '-..-.', '(': '-.--.', ')': '-.--.-',
  '&': '.-...', ':': '---...', ';': '-.-.-.', '=': '-...-', '+': '.-.-.', '-': '-....-',
  '_': '..--.-', '"': '.-..-.', '$': '...-..-', '@': '.--.-.', ' ': '/'
}
```

**Need**:
`src/utils/morse.js` file that have two function call `encode(...)` and `decode(...)`.

<details>
<summary>Function <code>encode(...)</code></summary>

```js
function encode(text: string): string
```

- This function have one string parameter call `text` that use to receive text from user.
- This function must returns morse code that encoded from `text` parameter.

</details>

<details>
<summary>Function <code>decode(...)</code></summary>

```js
function decode(morse: string): string
```

- This function have one string parameter call `morse` that use to receive morse code from user.
- This function must returns morse code that encoded from `morse` parameter

</details>

**Optional**:
- You can use `JSDoc` to describe your function. [Learn JSDoc](https://jsdoc.app/)

### PR DESCRIPTION
**Pull request by AkaKitsune027**
Translator function is done. **Please review !**

- encode function
- decode function

### CODE REVIEW

File: src/utils/morse.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export function encode(text) {
+    const upperCaseText = text.toUpperCase()
+    const result = []
+    for (const c of upperCaseText) {
+        result.push(morseList[c])
+    }
+    return result.join(' ')
+}
+
+/**
```
Comment (6MA-606) [inline src/utils/morse.js:26]: Your JSDoc is so good!

File: src/utils/morse.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    A: '.-', B: '-...', C: '-.-.', D: '-..', E: '.', F: '..-.', G: '--.', H: '....',
+    I: '..', J: '.---', K: '-.-', L: '.-..', M: '--', N: '-.', O: '---', P: '.--.',
+    Q: '--.-', R: '.-.', S: '...', T: '-', U: '..-', V: '...-', W: '.--', X: '-..-',
+    Y: '-.--', Z: '--..', 0: '-----', 1: '.----', 2: '..---', 3: '...--', 4: '....-',
+    5: '.....', 6: '-....', 7: '--...', 8: '---..', 9: '----.', '.': '.-.-.-', ',': '--..--',
+    '?': '..--..', "'": '.----.', '!': '-.-.--', '/': '-..-.', '(': '-.--.', ')': '-.--.-',
+    '&': '.-...', ':': '---...', ';': '-.-.-.', '=': '-...-', '+': '.-.-.', '-': '-....-',
+    '_': '..--.-', '"': '.-..-.', '$': '...-..-', '@': '.--.-.', ' ': '/'
+}
+
```
Comment (6MA-606) [inline src/utils/morse.js:11]: Can you add JSDoc for `encode` function?

File: src/utils/morse.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * This function use to convert morse code to text.
+ * @param {string} morse just input
+ * @returns 
+ */
+export function decode(morse) {
+    const morses = morse.split(' ')
+    console.log(morse);
+    let result = ''
+
```
Comment (6MA-606) [inline src/utils/morse.js:34]: Delete this line.

File: src/views/HomeView.vue [hunk 4]
```diff
@@ -1,15 +1,16 @@
 <script setup>
 import { ref, computed } from 'vue'
+import { encode, decode } from '../utils/morse.js'
 
-const mode = ref('encode')
+const mode = ref('encode','decode')
```
Comment (6MA-606) [inline src/views/HomeView.vue]: Oh, you don't need to edit this line. Please undo it!

File: src/views/HomeView.vue [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
 const input = ref('')
 const output = computed(() => {
   //TODO: Implement the morse code translation
```
Comment (6MA-606) [inline src/views/HomeView.vue]: Remove this comment, please.

**Review by 6MA-606 — CHANGES_REQUESTED** [review]
I already add my suggestion and comment to you. Please edit and request review again.

**Review by AkaKitsune027 — COMMENTED** [review]
I already fixed, Thanks ! **Please review again !**

**Review by 6MA-606 — APPROVED** [review]
OK, All green! You can **merge** now.

**Pull request merged**